### ISSUE TITLE
sphinx_gallery/tests/test_full.py::test_binder_logo_exists fails (path is clearly wrong)

### ISSUE THREAD

**Issue by mgorny** [sphinx-gallery/sphinx-gallery#746]
```
=============================================================== test session starts ===============================================================
platform linux -- Python 3.6.12, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
cachedir: .tox/py36/.pytest_cache
rootdir: /tmp/sphinx-gallery, configfile: setup.cfg
plugins: cov-2.10.1
collected 165 items                                                                                                                               

sphinx_gallery/tests/test_backreferences.py .......                                                                                         [  4%]
sphinx_gallery/tests/test_binder.py ..                                                                                                      [  5%]
sphinx_gallery/tests/test_docs_resolv.py ...                                                                                                [  7%]
sphinx_gallery/tests/test_full.py ......................F                                                                                   [ 21%]
sphinx_gallery/tests/test_gen_gallery.py ............................                                                                       [ 38%]
sphinx_gallery/tests/test_gen_rst.py .........................................................                                              [ 72%]
sphinx_gallery/tests/test_load_style.py .                                                                                                   [ 73%]
sphinx_gallery/tests/test_notebook.py ..............                                                                                        [ 81%]
sphinx_gallery/tests/test_py_source_parser.py ...............                                                                               [ 90%]
sphinx_gallery/tests/test_scrapers.py ..s......                                                                                             [ 96%]
sphinx_gallery/tests/test_sorting.py .                                                                                                      [ 96%]
sphinx_gallery/tests/test_sphinx_compatibility.py ...                                                                                       [ 98%]
sphinx_gallery/tests/test_utils.py ..                                                                                                       [100%]

==================================================================== FAILURES =====================================================================
_____________________________________________________________ test_binder_logo_exists _____________________________________________________________

sphinx_app = <sphinx.application.Sphinx object at 0x7f99a44b46d8>

    def test_binder_logo_exists(sphinx_app):
        """Check that the binder logo path is correct."""
        src_dir = sphinx_app.srcdir
        binder_logo = os.path.join(glr_path_static(), 'binder_badge_logo.svg')
        binder_logo_path = os.path.relpath(binder_logo, src_dir)
    
>       assert op.isfile('/{}'.format(binder_logo_path))
E       AssertionError: assert False
E        +  where False = <function isfile at 0x7f99ca7b69d8>('/../../../sphinx-gallery/sphinx_gallery/_static/binder_badge_logo.svg')
E        +    where <function isfile at 0x7f99ca7b69d8> = op.isfile
E        +    and   '/../../../sphinx-gallery/sphinx_gallery/_static/binder_badge_logo.svg' = <built-in method format of str object at 0x7f99c4b82d50>('../../../sphinx-gallery/sphinx_gallery/_static/binder_badge_logo.svg')
E        +      where <built-in method format of str object at 0x7f99c4b82d50> = '/{}'.format

sphinx_gallery/tests/test_full.py:824: AssertionError
```

This is on Gentoo Linux, reproduced with git master (839c02b0b076f20ea7cc0ca57a619ac7f00467f5). I don't know whether the relative path is correct but prepending it with `/` certainly isn't.

**Comment by mgorny** [comment]
I've used the following `tox.ini` to reproduce the issue reliably:

```
[tox]
envlist = py36,py37,py38,py39

[testenv]
deps =
	pytest
	pytest-cov
	sphinx
	matplotlib
	pillow
	joblib
commands = pytest
```

**Comment by lucyleeow** [comment]
Thanks for your report. It's related to #745 and we are probably going to change how we link it.

**Comment by lucyleeow** [comment]
closed by #745
